get_coverage_for_challenge.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+CHALLENGE_ID=$1
+RUBY_TEST_REPORT_CSV_FILE="${SCRIPT_CURRENT_DIR}/coverage/results.csv"
+RUBY_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/coverage.tdl"
+
+( cd ${SCRIPT_CURRENT_DIR} && \
+    $HOME/.rbenv/shims/bundle install && \
```

Comment:
This is a blocker.
The should not know of the existence of Rbenv.   It is the responsibility of the caller to have a working environment. It should use the executable from the PATH: `bundle` and `rake`.

Reply:
I agree with that, and I noticed when I ran the docker container manually via CLI, I could /bin/bash into it and run the commands without needing to use fullpath.

Although if I can the script that ran the container, the commands failed so I used fullpath. Its possible that the .bashrc which loads the path isn't getting loaded when run via the script. I will have to debug this.

Reply:
It could be, but the PATH should be configured externally.

Reply:
Will look into this asap